### PR TITLE
fix: fixed styling and update footer

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
Fixed footer and linked accounts and documentation site added

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -836,7 +834,7 @@ body {
     height: 151px;
 }
 
-.footer {
+footer {
     padding: 1rem;
     font-size: large;
```
Comment (hussiantabha) [inline css/style.css:835]: use rems

**Comment by hussiantabha** [comment]
Looks clean and good

**Review by Omkar-Chaskar — APPROVED** [review]
As @hussiantabha say . use rem

File: css/style.css [hunk 2]
```diff
@@ -836,7 +834,7 @@ body {
     height: 151px;
```
Comment (atharvabhanage02) [inline css/style.css]: Use rems here

File: css/style.css [hunk 3]
```diff
@@ -877,9 +875,11 @@ body {
...
     }
-    .nav-list{
-        padding:0.2rem;
+
+    .nav-list {
+        padding: 0.2rem;
     }
+
     .nav .nav-list li {
         padding: 0.5rem 0.2rem;
```
Comment (atharvabhanage02) [inline css/style.css:880]: You can use px here if the padding is too small . rem's can be used if the unit is anything above 10px

**Review by atharvabhanage02 — COMMENTED** [review]
Suggested few changes , please go through them

**Review by atharvabhanage02 — APPROVED** [review]

File: css/style.css (hunk 3 above)
Comment (Kothalkarpriya) [reply css/style.css:880]: Ok

File: css/style.css (hunk 2 above)
Comment (Kothalkarpriya) [reply css/style.css]: I will change it

File: css/style.css (hunk 1 above)
Comment (Kothalkarpriya) [reply css/style.css:835]: I will use it

**Comment by hussiantabha** [comment]
Looks good

**Pull request merged**